### DISCUSSION TITLE
Emit Filters to another component

### DISCUSSION THREAD

**Discussion by JimKarvo** [rappasoft/laravel-livewire-tables#1669] [Q&A]
Hello,

I need to update an html element before the table, and I want to know the table's filters, in order to have accurate data.

I can;t find any function for emit the filters to another component, on filter change

any ideas?

**Comment by lrljoe** [comment]
Can you expand a bit on what exactly you want to extract out/emit to another component please?

There are a few different options, but it depends exactly what you're trying to pull out!

I'm trying to avoid emitting events throughout, but it wouldn't be an issue to add a Lifecycle Hook at the applying a filter stage.

You could also over-ride the core updated hooks, so long as you call back to the parent, in the interim.

More info will help me to figure out the best route for you
